woocommerce/woocommerce-android
# Refund by items: Various refactorings - Step 3

This PR is a mix of different things:

- The refund items layout is moved into a separate file so it can be reused in the refund details screen
- The number-picker dialog is refactored so that the navigation component is used to display it
- The `View.expand()` function is updated to fix the smoothness of the animation
- The Shipping refund section stub is added (I've enabled it just for this PR to test the expand/collapse functionality; the shipping refund has been postponed until it's supported by the API)

To test:

1. Go to order detail and issue a refund
2. Notice there is a shipping refund toggle
3. Tap on the toggle and enjoy the beautiful expanding animation (no data is filled out)
4. Tap the toggle button again and watch the collapse animation

~~**Note:** #1622 must be merged before this!~~
